packages/router-plugin/src/core/code-splitter/compilers.ts

```diff
@@ -576,7 +597,7 @@ export function compileCodeSplitVirtualRoute(
       return str
     }, '')
 
-    const warningMessage = `These exports from "${opts.filename.replace('?' + splitPrefix, '')}" are not being code-split and will increase your bundle size: ${list}\nThese should either have their export statements removed or be imported from another file that is not a route.`
+    const warningMessage = `These exports from "${opts.filename}" are not being code-split and will increase your bundle size: ${list}\nThese should either have their export statements removed or be imported from another file that is not a route.`
```

Comment:
Since we now can have more than one virtual file per-route, leaving in the `?tsr-split-***` value is useful for debugging.